libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1215,7 +1218,7 @@ namespace SohImGui {
                 EnhancementCheckbox("Fireproof Deku Shield", "gFireproofDekuShield");
                 Tooltip("Prevents the Deku Shield from burning on contact with fire");
                 EnhancementCheckbox("Shield with Two-Handed Weapons", "gShieldTwoHanded");
-                Tooltip("Allows Link to shield normally with two-handed swords and the Megaton Hammer");
+                Tooltip("This allows you to put up your shield with any two-handed weapon in hand");
```

Comment:
This one is not quite correct, as the cheat explicitly does *not* apply to Deku Sticks, even though they are also a two-handed weapon.